spk/flexget/Makefile

```diff
@@ -1,16 +1,18 @@
 SPK_NAME = flexget
-SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 5
+SPK_VERS = 2.9.14
+SPK_REV = 6
 SPK_ICON = src/flexget.png
 
+# Python wheels
+# https://github.com/SynoCommunity/spksrc/wiki/Using-wheels-to-distribute-Python-packages
```

Comment:
In fact my fault, I would have read carefully https://github.com/SynoCommunity/spksrc/wiki/Developers-HOW-TO before getting into that task